common/environment/setup/install.sh

```diff
@@ -98,6 +98,17 @@ _vman() {
 	return 1
 }
 
+_vdesktop() {
+	local file="$1" targetfile="$2"
+
+	if [ $# -lt 1 ]; then
+		msg_red "$pkgver: vdesktop: 1 argument expected: <file>\n"
+		return 1
+	fi
+
+	vinstall "$file" 644 "usr/share/applications/${pkgname}" "$targetfile"
```

Comment:
> .desktop files go directly into `usr/share/applications`, not per package subdirs...

Got it

> Also you'd want to use unquoted `$targetfile`, otherwise it will be passing an empty string as last arg and that's entirely wrong.

I thought so too, but the other helpers (`vbin`, `vdoc`, `vconf`, `vsconf`, `vlicense`) seemed to be working fine.